modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/rate/DispatchingRateComputationFn.java

```diff
@@ -246,6 +269,12 @@ public double explainRate(
       builder.put(ExplainKey.FIXED_RATE, rate);
       builder.put(ExplainKey.COMBINED_RATE, rate);
       return rate;
+    } else if (computation instanceof FixedOvernightCompoundedAnnualRateComputation) {
+      // inline code (performance) avoiding need for FixedRateComputationFn implementation
+      double rate = ((FixedOvernightCompoundedAnnualRateComputation) computation).getRate();
+      builder.put(ExplainKey.FIXED_RATE, rate);
+      builder.put(ExplainKey.COMBINED_RATE, rate);
```

Comment:
Should the COMBINED_RATE be the result of `getSimpleRate`?

Reply:
Yes I was wondering about this. The simple rate or the accrued rate?

Reply:
FIXED_RATE is the "normal" rate in the contract, COMBINED_RATE is the rate after processing (gearing, spread etc.)